apps/agentfabric/config_utils.py

```diff
@@ -11,33 +11,66 @@
 DEFAULT_TOOL_CONFIG_FILE = './config/tool_config.json'
 
 
-def save_builder_configuration(builder_cfg):
+def get_user_file_name(uuid_str=''):
     builder_cfg_file = os.getenv('BUILDER_CONFIG_FILE',
                                  DEFAULT_BUILDER_CONFIG_FILE)
+    # convert from ./config/builder_config.json to ./config/builder_config_uuid.json
+    if uuid_str != '':
```

Comment:
aggregate files of one user to one specific user config dir, rather than using uuid as suffix